kala.py

```diff
@@ -73,6 +73,9 @@ def get(mongodb, collection):
     if distinct:
         return {'values': cursor.distinct(distinct)}
 
+    if bottle.request.query.get('count'):
```

Comment:
I think I'd personally use `if 'count' in bottle.request.query` unless there's any reason not to, certain to be testing existence and not value then, thinking of the case if someone provides `?count`, which would return `''`.

Otherwise looks fine!